jobs/config.json

```diff
@@ -6341,6 +6341,26 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gke-staging-latest-device-plugin-gpu": {
+    "args": [
+      "--check-leaked-resources",
+      "--deployment=gke",
+      "--extract=gs://kubernetes-release-gke/release/v1.8.2-gke.0",
```

Comment:
you can use `--extract=ci/gke-latest-staging` now, as https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L4980

Reply:
I'll update other jobs

Reply:
Done.